index.html

```diff
@@ -4,15 +4,86 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    
     <!-- Add a link to your custom CSS here -->
 
+    <link rel="stylesheet" href="css/main.css">
     <title>Responsive grid project</title>
 </head>
 
 <body>
 
     <!-- Add your markup here -->
+<main class="h-main">
 
+     <section class="h-hero">
+       <h2>
+        <input type="checkbox">Maya Nelson</h2>
+       <p>PROJECT MANAGER</p>
+     </section>
+
+     <section class="h-feature">
+        
+          <img src="maya.jpeg" alt="maya">
+          <h2>Maya</h2>
+          <h2 >Nelson</h2>
+          <button class="one"><a id="left" href="#">RESUME</a></button> <button class="two" ><a id="right" href="#">PROJECTS</a></button><br>
+        
+        <div class="logo">
+          <img src="facebook-icon.svg" alt="facebook" width="20">
+          <img src="twitter-icon.svg" alt="twitter" width="20">
+          <img src="linkedIn.png" alt="linkedin" width="20">
+          <img src="instagram-icon.svg" alt="instagram" width="20">
+        </div>
+     </section>
+
+     <section class="h-products">
+       <div class="text"> 
+        <h1>HELLO</h1>
+        <h3>Here's who I am & what I do</h3>
+        <p>
+            I'm a paragraph. Click here to add<br> 
+            your own text and edit me. It's easy.<br>
+            Just click "Edit Text" or double click<br> 
+            me to add your own content and <br>
+            make changes to the font.
+        </p>
+        <p>
+            I'm a great place for you to tell a <br>
+            story and let your users know a little<br>
+            more about you.
+        </p>
+       </div>
+     </section>
+
+        <section class="h-aside">
+          <div class="h-up">
+            <h3>Call</h3>   
+            <h3>Write</h3>
+          </div>
+          <div class="h-down">
+            <p>123-456-7890</p>
```

Comment:
Could this be a +tel link?

Reply:
There's no a +tel link as per original design.

Reply:
I think it could be a tel link! 😉